compiler/rustc_mir_build/src/build/scope.rs

```diff
@@ -1021,6 +1023,38 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
         cached_drop
     }
 
+    /// This is similar to [diverge_cleanup_target] except its target is set to
```

Comment:
Should this be `[diverge_cleanup]`?
```suggestion
    /// This is similar to [diverge_cleanup] except its target is set to
```